src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$width-boxs: 100px; // Ширина одного дня
+$width-gap: 1px; // Проміжок між днями
+$calendar-days: 31; // Максимальна кількість днів у місяці
+$days-of-the-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+); // Відображення днів тижня
+$my-var: $width-boxs * 7 + 6 * $width-gap; // Загальна ширина календаря (7 колонок з урахуванням проміжків)
```

Comment:
The variable `$my-var` does not have a semantic name. It is important to name variables in a way that describes their purpose or the value they hold for better readability and maintainability of the code.